### DISCUSSION TITLE
Minoston MP31Z reports wrong product id <informational>

### DISCUSSION THREAD

**Discussion by drizuid** [zwave-js/zwave-js#5290] [General]
I was not sure where the best place to put this was. The mp31z is much like the mp21z but shorter rectangle and zwave 700. Unfortunately, they did not update their productid in the device, so it is the same as the mp21z which causes zwave js ui to reflect it as such. I have emailed Minoston today (they tend to be pretty responsive in my experience) about this, but I wanted to ensure I shared with my fellow zwavejs fans. The device works fine from what i can tell and I notice no missing functions.

<img width="422" alt="image" src="https://user-images.githubusercontent.com/40674481/210250164-8a6f802c-dc9d-495c-9445-7c5ac7d3590f.png">

**Comment by drizuid** [comment]
response from vendor:
<img width="411" alt="image" src="https://user-images.githubusercontent.com/40674481/210288228-0da300dd-3949-418f-8005-83611390d3c5.png">

I'll follow up if they say anything else or perhaps make changes in future models.

**Reply by AlCalzone** [reply]
So they did not change config params? I thought the 31 had different ones. If they are truly identical, we can just update the label.

**Reply by drizuid** [reply]
There's always a chance I don't know what I'm doing (strong chance in fact), but their reply does make it seem like it wasn't changed.  If the screenshots I show aren't the right way to know definitely then I'm probably incorrect. Is there anything I can provide that would be useful?
